packages/members-api/lib/MembersAPI.js

```diff
@@ -265,14 +266,26 @@ module.exports = function MembersAPI({
         return getMemberIdentityData(email);
     }
 
-    async function getMemberIdentityData(email) {
+    async function getMemberIdentityData(email, {include = ''} = {}) {
+        const withRelated = [
+            'stripeSubscriptions',
+            'stripeSubscriptions.stripePrice',
+            'products'
+        ];
+        const requestedQueryIncludes = include.split(',') || [];
+        const optionalIncludes = [
+            'labels',
+            'stripeSubscriptions.customer',
+            'stripeSubscriptions.stripePrice.stripeProduct',
+            'stripeSubscriptions.stripePrice.stripeProduct.product'
+        ];
```

Comment:
@allouis This change allows adding optional `?include=...` in the request for member data session, which allows fetching all the extra data that we don't need right now in future easily. I was wondering if it made more sense to force `include` for even the default data points like `products`/`subscriptions`, but that seemed a bit of an overkill, interested what you think about this approach